app/components-react/editor/elements/SourceSelector.tsx

```diff
@@ -632,6 +648,73 @@ class SourceSelectorController {
     return this.scene.getSelection(sceneNodeId);
   }
 
+  toggleDualOutput() {
+    if (this.userService.isLoggedIn) {
+      if (Services.StreamingService.views.isMidStreamMode) {
+        message.error({
+          content: $t('Cannot toggle Dual Output while live.'),
+          className: styles.toggleError,
+        });
+      } else if (Services.TransitionsService.views.studioMode) {
+        message.error({
+          content: $t('Cannot toggle Dual Output while in Studio Mode.'),
+          className: styles.toggleError,
+        });
+      } else {
+        const dualOutputToggleHandled = this.dualOutputService.dualOutputToggleHandled.subscribe(
+          (enabled: boolean) => {
+            console.log('enabled', enabled);
```

Comment:
dangling console, if we wanna keep it I think we should add some context as to what's enabled